cypress/integration/dib-travel/personal-settings/profile/profile.spec.ts

```diff
@@ -0,0 +1,139 @@
+    ).select(profileDetails.birthMonth);
+    cy.get(
+      '.cdk-overlay-container dib-travel-document-dialog dib-select-dob[ng-reflect-name="dateOfBirth"] select.year'
+    ).select(profileDetails.birthYear);
+    cy.get('dib-travel-document-dialog dib-input input[name="idNumber"]').type(profileDetails.documentNumber);
+    cy.get('dib-travel-document-dialog dib-input input[placeholder="Issuing country"]').type(
+      profileDetails.issuingCountry
+    );
+    cy.get('dib-travel-document-dialog dib-searchable-select div').contains(profileDetails.issuingCountry).click();
+    cy.get(
+      '.cdk-overlay-container dib-travel-document-dialog dib-select-dob[ng-reflect-name="expiryDate"] select.day'
+    ).select(profileDetails.expiryDay);
+    cy.get(
+      '.cdk-overlay-container dib-travel-document-dialog dib-select-dob[ng-reflect-name="expiryDate"] select.month'
+    ).select(profileDetails.expiryMonty);
+    cy.get(
+      '.cdk-overlay-container dib-travel-document-dialog dib-select-dob[ng-reflect-name="expiryDate"] select.year'
+    ).select(profileDetails.expiryYear);
+    cy.get('dib-travel-document-dialog dib-input input[placeholder="Nationality"]').type(profileDetails.nationality);
+    cy.get('dib-travel-document-dialog dib-searchable-select div').contains(profileDetails.nationality).click();
+    cy.get('dib-travel-document-dialog ui-checkbox-wrapper').click();
+    cy.get('dib-travel-document-dialog ui-button').click();
+    cy.get('dib-profile dib-account dib-travel-documents').should('contain', 'PASSPORT');
+    cy.get('dib-profile dib-account dib-travel-documents').should('contain', profileDetails.issuingCountry);
+    cy.get('dib-account dib-travel-documents dib-simple-button.btn').click();
+    cy.get('confirmation-dialog ui-button.button.ng-star-inserted').click();
+  });
+
+  it('should add loyalty program and delete it', () => {
+    cy.get('dib-profile  dib-account  div.profile-info__section.loyalty-program button').click();
+    cy.get('dib-add-loyalty button').click();
+    cy.get('dib-add-loyalty dib-loyalty-auto-complete')
+      .type(profileDetails.loyaltyProgram)
+      .type('{downarrow}')
+      .type('{enter}');
+    cy.get('dib-dialog-wrapper dib-add-loyalty input[name="program"]').type(profileDetails.loyaltyNumber);
+    cy.get('dib-dialog-wrapper  dib-add-loyalty button').click();
+    cy.get('.cdk-overlay-container').should('contain', 'Loyalty program saved');
+    cy.get('dib-profile dib-account .loyalty-program__grid').should('contain', profileDetails.loyaltyProgram);
+    cy.get('dib-profile dib-account .loyalty-program__grid').should('contain', profileDetails.loyaltyNumber);
+    cy.get('dib-account  div.grid-button button').click();
+    cy.get('confirmation-dialog ui-button.button.ng-star-inserted').click();
+    cy.get('.cdk-overlay-container').should('contain', 'Loyalty program successfully deleted');
```

Comment:
Can we add at least one or ideally two more selectors here? Apply similar in all other places for `.cdk-overlay-container`